src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const MAX = 10;
+const MIN = 2;
```

Comment:
MAX and MIN what?) In this case it's ok, but in future your variables should describe them better